content/applications/finance/accounting/fiscal_localizations/localizations/germany.rst

```diff
@@ -31,7 +31,135 @@ Export from Odoo to Datev
 It is possible to export your accounting entries from Odoo to Datev. To be able to use this
 feature, the german accounting localization needs to be installed on your Odoo Enterprise database.
 Then you can go in :menuselection:`Accounting --> Reporting --> General Ledger` then click on the
-**Export Datev (csv)** button. 
+**Export Datev (csv)** button.
+
+.. _germany/pos:
+
+Point of Sale in Germany: Technical Security System
+===================================================
+
+The **Kassensicherungsverordnung** (The Act on Protection against Manipulation of Digital Records)
+requires that electronic record-keeping systems - including the :doc:`point of sale
+</applications/sales/point_of_sale>` systems - must be equipped with a **Technical Security System**
```

Comment:
It's usually best to write relative paths than absolute paths when you don't need to go up to the root (both pages are under `applications`). It makes it easier to make changes to the structure of their common ancestors. In this case, however, the chain of `../` required for a relative link might need to be regularly updated to follow the frequent restructurations made under the `accounting` directory. So I'll let you judge whether an absolute or a relative link is more appropriate here, depending on whether new structural changes are planned for this directory.